fs/io_zenfs.cc

```diff
@@ -705,6 +713,36 @@ void ZoneFile::ReplaceExtentList(std::vector<ZoneExtent*> new_list) {
   extents_ = new_list;
 }
 
+void ZoneFile::SetLinkName(const std::string& linkf) {
+  linkfiles_.push_back(linkf);
+}
+
+IOStatus ZoneFile::RenameLink(std::string src, std::string dest) {
+  if (GetFilename() == src) {
```

Comment:
It does not make sense to me to keep rename() treating the first name in the list special. Removing rename(), we can simplify this function..

Reply:
Agreed, we can just remove the `if` part of this branch and only keep the `else` part.